models/eos/static_routes/eos_static_routes.yml

```diff
@@ -0,0 +1,120 @@
+  description: This module configures and manages the attributes of static routes on Arista EOS platforms.
+  author: Gomathi Selvi Srinivasan (@GomathiselviS)
+  notes:
+  options:
+    config:
+      description:
+        - A list of configurations for static routes
+      type: list
+      elements: dict
+      suboptions:
+        vrf:
+          description:
+            - The VRF to which the static route(s) belong
+          type: str
+          default: __global__
+        address_families:
+          description: A dictionary specifying the address family to which the static route(s) belong.
+          type: list
+          elements: dict
+          suboptions:
+            afi:
+              description:
+                - Specifies the top level address family indicator.
+              type: str
+              choices: ['ipv4', 'ipv6']
+              required: True
+            routes:
+              description: A dictionary that specifies the static route configurations
+              elements: dict
+              type: list
+              suboptions:
+                dest:
+                  description: 
+                    - Destination IPv4 subnet (CIDR or address-mask notation)
+                    - The address format is <v4/v6 address>/<mask> or <v4/v6 address> <mask>
+                    - The mask is number in range 0-32 for IPv4 and in range 0-128 for IPv6
+                  type: str
+                  required: True
+                next_hops: 
+                  description: 
+                    - Details of route to be taken
+                  type: list
+                  elements: dict
+                  suboptions:
+                    forward_router_address:
+                      description:
+                        - Forwarding router's address on destination interface
+                      type: str
+                    interface:
+                      description:
+                        - Outgoing interface to take. For anything except 'null0', then next hop IP address should also be configured.
+                        - IP address of the next hop router or
+                        - null0 Null0 interface or
+                        - ethernet e_num Ethernet interface or
+                        - loopback l_num Loopback interface or
+                        - management m_num Management interface or
+                        - port-channel p_num
+                        - vlan v_num
+                        - vxlan vx_num
+                        - Nexthop-Group  Specify nexthop group name
+                        - Tunnel  Tunnel interface
+                        - vtep  Configure VXLAN Tunnel End Points
+                      type: str
+                    admin_distance:
+                      description: 
+                        - Preference or administrative distance of route (range 1-255)
+                      type: int
+                    route_name:
```

Comment:
Is `route_name` basically a description for the route? If yes, we have used the `description` key to signify it.